ext/SolidStateDetectorsLegendHDF5IOExt.jl

```diff
@@ -2,7 +2,7 @@
 
 module SolidStateDetectorsLegendHDF5IOExt
 
-import LegendHDF5IO
+import ..LegendHDF5IO
```

Comment:
@oschulz In the old code, `LegendHDF5IO` had to be imported with `..` before the package name if it was loaded via `Requires`.
Can you confirm that this is correct and needed?

Reply:
`import ..LegendHDF5IO` seems to be the recommended way.